### PR TITLE
Questions score page

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
Done - Questions Page and Score Page

### CODE REVIEW

File: src/Components/Navbar/navbar.js [hunk 1]
```diff
@@ -18,11 +18,8 @@ const Navbar = () => {
       <ul className="header-nav">
         <li style={{ marginTop: "0.25rem", cursor: "pointer" }}>
           <li style={{ cursor: "pointer" }}>
-            <a>
-              <i
-                class="fas fa-sign-out-alt"
-                style={{ marginRight: "0.5rem" }}
-              ></i>
+            <a href="https://github.com/MuskanChhatrasal">
```
Comment (Developers-codz) [inline src/Components/Navbar/navbar.js:21]: for external link can add target _blank property

**Review by Developers-codz — APPROVED** [review]
Code is readable UI is also good
You can merge for sure

File: src/Pages/Questions/Questions.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          </Button>
+        </Box>
+      ))}
+      <Box mt={5}>
+        Score: {score} / {response.results.length}
+      </Box>
+      <img
+        src={Question}
+        alt="questions"
+        style={{ marginLeft: "55rem", marginTop: "-25rem" }}
```
Comment (MayankKumar10) [inline src/Pages/Questions/Questions.js:107]: Suggestion: Try to avoid using inline styling

**Review by MayankKumar10 — APPROVED** [review]
Look into this suggestion everything else looks good, Nice UI you can merge it

**Pull request merged**